### DISCUSSION TITLE
Multiple users simultaneously using different windows desktops seperately

### DISCUSSION THREAD

**Discussion by webclinic017** [rustdesk/rustdesk#4983] [Feature Request] [answered]
Can this feature be implemented, where

single windows systems can be shared with multiple users and each user has access to only one desktop view - Virtual Desktops whicha re found in task bar.. technically isolating each user activity?

**Comment by ghost** [comment]
This requires terminal services on the windows machine. Only Windows Server software has this.

**Comment by BennyBeat** [comment, marked as answer]
@webclinic017 
Perhaps you're interested in this service:
https://github.com/sebaxakerhtc/rdpwrap/releases

Hope this can help you!

**Reply by webclinic017** [reply]
Benny thanks alot - I got to install it and this video also helped https://www.youtube.com/watch?v=vtJ4Uqsa8fA - I could have this now working..
